rep-0008.rst

```diff
@@ -67,12 +133,12 @@ Module with msgs/srvs::
           |- yourfiles.py
      |- scripts/
         |- non-exported python files
-    
+
 We distinguish between "nodes" and "scripts" for clarity to
 users. Nodes are executable Python files that conform to the ROS node
 API. Scripts are executable Python files that do not conform to the
 ROS node API.
```

Comment:
Principally, I think this documentation describes what should be in the rep, w.r.t installing scripts:

http://docs.ros.org/api/catkin/html/howto/format2/installing_python.html